src/main/java/io/anserini/qa/passage/ScoredPassage.java

```diff
@@ -0,0 +1,30 @@
+package io.anserini.qa.passage;
+
+public class ScoredPassage implements Comparable<ScoredPassage> {
+  String sentence;
+  double score;
+
+  public ScoredPassage(String sentence, double score) {
+    this.sentence = sentence;
+    this.score = score;
+  }
+
+  public String getSentence() {
+    return  sentence;
+  }
+
+  public double getScore() {
+    return  score;
+  }
```

Comment:
nah, this is good - actually, it's IntelliJ that puts short methods on a single line...